test/tools/libtesteth/ImportTest.cpp

```diff
@@ -260,23 +260,30 @@ std::tuple<eth::State, ImportTest::ExecOutput, eth::ChangeLog> ImportTest::execu
 	return std::make_tuple(initialState, out, initialState.changeLog());
 }
 
+//convert json object of "key" : "stringdeciamal"  into "key" : "stringhex"
 json_spirit::mObject& ImportTest::makeAllFieldsHex(json_spirit::mObject& _o, bool _isHeader)
 {
 	static const set<string> hashes {"bloom" , "coinbase", "hash", "mixHash", "parentHash", "receiptTrie",
 									 "stateRoot", "transactionsTrie", "uncleHash", "currentCoinbase",
-									 "previousHash", "to", "address", "caller", "origin", "secretKey", "data", "extraData"};
-
+									 "previousHash", "to", "address", "caller", "origin", "secretKey", "extraData"};
```

Comment:
Yes, we have a choice to define `string const g_bloom = "bloom"` and use `g_bloom` here.  I'm suggesting to do that.